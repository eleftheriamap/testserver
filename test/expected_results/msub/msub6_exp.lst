
test/exp/test/gen/msub/msub6_exp.bin:     file format binary


Disassembly of section .data:

0000000000000000 <.data>:
   0:	5105877b 	sub	w27, w27, #0x161
   4:	9151a884 	add	x4, x4, #0x46a, lsl #12
   8:	115082d6 	add	w22, w22, #0x420, lsl #12
   c:	1b04db69 	msub	w9, w27, w4, w22
  10:	8a000000 	and	x0, x0, x0
