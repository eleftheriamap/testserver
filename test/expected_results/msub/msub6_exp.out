Registers:
X00    = 0000000000000000
X01    = 0000000000000000
X02    = 0000000000000000
X03    = 0000000000000000
X04    = 000000000046a000
X05    = 0000000000000000
X06    = 0000000000000000
X07    = 0000000000000000
X08    = 0000000000000000
X09    = 0000000061a4a000
X10    = 0000000000000000
X11    = 0000000000000000
X12    = 0000000000000000
X13    = 0000000000000000
X14    = 0000000000000000
X15    = 0000000000000000
X16    = 0000000000000000
X17    = 0000000000000000
X18    = 0000000000000000
X19    = 0000000000000000
X20    = 0000000000000000
X21    = 0000000000000000
X22    = 0000000000420000
X23    = 0000000000000000
X24    = 0000000000000000
X25    = 0000000000000000
X26    = 0000000000000000
X27    = 00000000fffffe9f
X28    = 0000000000000000
X29    = 0000000000000000
X30    = 0000000000000000
PC     = 0000000000000010
PSTATE : -Z--
Non-zero memory:
0x00000000 : 0x5105877b
0x00000004 : 0x9151a884
0x00000008 : 0x115082d6
0x0000000c : 0x1b04db69
0x00000010 : 0x8a000000
